docs/content/start/telemetry.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Quick Start
+menuPosition: 4
+codeCopyButton: true
+aliases:
+  - "/docs/get-started/telemetry/"
+---
+
+
+With thousands of developers using our framework to build their services and applications, telemetry is an essential part of maintaining and troubleshooting the health of the applications, so we are offering just that! We have integrated a way for the developers to set up a telemetry system by allowing them to create custom logging and handling logics to be passed in and called by our Fluid Framework telemetry pipeline.
+
+## ITelemetryBaseLogger interface
+
+The `ITelemetryBaseLogger` is an interface within the `@fluidframework/common-definitions` package. This interface can be implemented and passed into the `createContainer()` and `getContainer()` methods in `TinyliciousClient` as part of the `TinyliciousContainerConfig` type parameter.
+
+```ts
+export interface TinyliciousContainerConfig {
+  id: string;
+  logger?: ITelemetryBaseLogger;
+}
+```
+
+```ts
+public async createContainer(
+  serviceContainerConfig: TinyliciousContainerConfig,
+  containerSchema: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
```

Comment:
I don't know how the docs are presenting the "front door" for devs, but if this section could be made more generic that would be great. Like, if it's not too "bare" of an API to speak of, the real entry point here is in the constructor of `Loader` and `ILoaderProps.logger`. I'd like to see that mentioned here, unless we're really directing people only to the `FooClient`s, in which case I'd present tinylicious as an example, rather than assuming that context.

Reply:
good point. I elaborated it to include `Loader`. Exposing the users to the APIs will definitely give them a better understanding of how it works under the hood.